### PR TITLE
[fast-reboot] Back up FDB/ARP/Default routes

### PR DESCRIPTION
**Pull request by tahmed-dev**
FDB/ARP/Default routes files are deleted after swssconfig. This
make debugging/validation of device conversion hard. This PR
save those files in order to facilitate device conversion.

signed-off-by: Tamer Ahmed <tamer.ahmed@microsoft.com>

**- Why I did it**
In order to make verification of device conversion easier.

**- How I did it**
Code change

**- How to verify it**
Verified with debug build that has this change
```
admin@str-s6000-acs-14:~$ sudo find / -name arp.json*
/host/image-taahme_backup-file-during-fast-reboot.0-dirty-20200618.235059/docker/overlay2/522da426044901997983d9c88f77fa14f123f48e2038af11f13bd357123c8c77/diff/arp.json.1
/usr/lib/python2.7/dist-packages/sonic-utilities-tests/filter_fdb_input/arp.json
/var/lib/docker/overlay2/522da426044901997983d9c88f77fa14f123f48e2038af11f13bd357123c8c77/diff/arp.json.1
/var/lib/docker/overlay2/522da426044901997983d9c88f77fa14f123f48e2038af11f13bd357123c8c77/merged/arp.json.1
admin@str-s6000-acs-14:~$ sudo find / -name default_routes.json*
/host/image-taahme_backup-file-during-fast-reboot.0-dirty-20200618.235059/docker/overlay2/522da426044901997983d9c88f77fa14f123f48e2038af11f13bd357123c8c77/diff/default_routes.json.1
/var/lib/docker/overlay2/522da426044901997983d9c88f77fa14f123f48e2038af11f13bd357123c8c77/diff/default_routes.json.1
/var/lib/docker/overlay2/522da426044901997983d9c88f77fa14f123f48e2038af11f13bd357123c8c77/merged/default_routes.json.1
admin@str-s6000-acs-14:~$ sudo find / -name fdb.json*
/host/image-taahme_backup-file-during-fast-reboot.0-dirty-20200618.235059/docker/overlay2/522da426044901997983d9c88f77fa14f123f48e2038af11f13bd357123c8c77/diff/fdb.json.1
/usr/lib/python2.7/dist-packages/sonic-utilities-tests/filter_fdb_input/fdb.json
/var/lib/docker/overlay2/522da426044901997983d9c88f77fa14f123f48e2038af11f13bd357123c8c77/diff/fdb.json.1
/var/lib/docker/overlay2/522da426044901997983d9c88f77fa14f123f48e2038af11f13bd357123c8c77/merged/fdb.json.1
```

**- Description for the changelog**

### CODE REVIEW

**Comment by tahmed-dev** [comment]
retest mellanox please

**Comment by lguohan** [comment]
@tahmed-dev, let me know if you have validated so that we can merge.

File: dockers/docker-orchagent/swssconfig.sh [hunk 1]
```diff
@@ -8,19 +8,19 @@ function fast_reboot {
       if [[ -f /fdb.json ]];
       then
         swssconfig /fdb.json
-        rm -f /fdb.json
+        mv /fdb.json /fdb.json.1
```
Comment (yxieca) [inline dockers/docker-orchagent/swssconfig.sh:11]: mv -f? the .1 file might exist with unexpected permission.

Comment (tahmed-dev) [reply dockers/docker-orchagent/swssconfig.sh:11]: Ack

**Comment by tahmed-dev** [comment]
> @tahmed-dev, let me know if you have validated so that we can merge.

Ack, will update the PR once validation is complete.

**Comment by lguohan** [comment]
why we are having this build error? https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-vs-image-pr/4406/

**Comment by lguohan** [comment]
retest vsimage please

**Comment by tahmed-dev** [comment]
> why we are having this build error? https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-vs-image-pr/4406/
> 
> does not seem like random.

Interesting! I see this error and it does not look like it is related to change in here. Will keep an eye on it
```
10:07:56  >       namespaces = sonic_device_util.get_all_namespaces()
10:07:56  E       AttributeError: 'module' object has no attribute 'get_all_namespaces'
```

**Comment by jleveque** [comment]
vsimage build appears to be stuck. Retest vsimage please.

**Comment by jleveque** [comment]
Retest vsimage please

**Pull request merged**